### PR TITLE
Confirm restore presentation

### PR DESCRIPTION
**Pull request by fjvallarino**
@Rise-Vision/apps please review

### CODE REVIEW

File: web/scripts/editor/services/svc-editor-factory.js [hunk 1]
```diff
@@ -321,8 +322,33 @@ angular.module('risevision.editor.services')
...
+          template: $templateCache.get(
+            'confirm-instance/confirm-modal.html'),
+          controller: 'confirmInstance',
+          windowClass: 'modal-custom',
+          resolve: {
+            confirmationTitle: function () {
+              return 'Restoring Presentation';
+            },
+            confirmationMessage: function () {
+              return 'Are you sure you want to Restore this Presentation to it\'s Published version? All changes will be lost.';
```
Comment (alex-deaconu) [inline web/scripts/editor/services/svc-editor-factory.js]: @fjvallarino wondering if you can move these 2 strings to common-i18n. Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino looks good, just one comment re: i18n strings. Thanks.

**Comment by fjvallarino** [comment]
@Rise-Vision/apps updated with common-i18n references

**Comment by alex-deaconu** [comment]
@fjvallarino I clicked the "Update branch" button and it restaged the i18n changes. PR looks good thanks for fixing 👍 .

**Pull request merged**